tests/org.eclipse.reddeer.eclipse.test/plugin.xml

```diff
@@ -17,6 +17,19 @@
    		<runtimeType id="org.eclipse.reddeer.eclipse.test.wst.server.testserverruntime" name="Test runtime" description="Testing runtime" facetRuntimeComponent="org.eclipse.wst.server.http.runtime" facetRuntimeVersion="1.0" class="org.eclipse.reddeer.eclipse.test.wst.server.ui.TestServerRuntime">
    			<moduleType types="PHP" versions="1.0"/>
    			<moduleType types="wst.web" versions="1.0"/>
+   			<moduleType types="jst.appclient" versions="5.0, 6.0,7.0,8.0"/>
+   			<moduleType types="wst.web" versions="1.0,1.2,1.3,1.4"/>
+   			<moduleType types="jst.web" versions="2.2, 2.3, 2.4, 2.5, 3.0,3.1,4.0"/>
+   			<moduleType types="jst.ejb" versions="1.0, 1.1, 2.0, 2.1, 3.0, 3.1,3.2"/>
+   			<moduleType types="jst.ear" versions="1.2, 1.3, 1.4, 5.0, 6.0,7.0,8.0"/>
+   			<moduleType types="jst.connector" versions="1.0, 1.5, 1.6,1.7"/>
+   			<moduleType types="jst.utility" versions="1.0"/>
+   			<moduleType types="jboss.package" versions="1.0"/>
+   			<moduleType types="jboss.singlefile" versions="1.0"/>
+   			<moduleType types="jst.jboss.sar" versions="1.0"/>
+   			<moduleType types="jbt.bpel.module" versions="1.1, 2.0"/>
+   			<moduleType types="bpel.module" versions="1.1, 2.0"/>
+   			<moduleType types="jboss.osgi" versions="1.0"/>
```

Comment:
You have just copied all capabilities from WildFly, please select just what you need.

Reply:
Removed what is not needed.